test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
I think you misunderstood my remark: I would add notes right here to state: "DPI needs to be set to a constant because we use that to scale some units", not something like "this requires DPI to be set to X" in the affected tests, because that will indeed be too much work.

What we gain, IMHO, from having this documented is that we will know in the future why we need that line of code. Also, the minimalism is something that I think will only help us: otherwise we can as well dump `get(0,'Factory')` and `get(0)` from a fixed release. Maybe that is a viable way, but that piece of code would be nothing more readable than a binary blob IMO and could make debugging quite hard.

But to answer your question: I would prefer if you remove it here (I'm sorry that requires you to regenerate the hashes), but to be honest: I will still merge it at one point or another, even if you don't.